tidb-in-kubernetes/aws-eks-tutorial.md

```diff
@@ -0,0 +1,177 @@
+# 在 AWS EKS 上部署 TiDB Operator 和 TiDB 集群
```

Comment:
This file should be put in docs-cn/how-to/get-started/in-kubernetes/ directory. Please see [TOC with TiDB in Kubernetes Integrated](https://docs.google.com/document/d/1k7zvGbcoMj1ggo28uZ5gr3-GXNiPAIWda7Df1oXOZbM/edit?usp=sharing) for details.